frontend/apps/web/app/(mgmt)/[account]/transformers/page.tsx

```diff
@@ -47,11 +50,16 @@ function TransformersTable(): ReactElement {
   if (transformersIsLoading || userDefinedTransformersLoading) {
     return <SkeletonTable />;
   }
-  const systemTransformerColumns = getSystemTransformerColumns();
+  const systemTransformerColumns = getSystemTransformerColumns({
+    accountName: '',
+    transformerSource: '',
```

Comment:
This can actually go away since the transformer data will be pulled from the row itself in the generate function.

Reply:
The whole thing cannot go away as it starting giving errors, props need to be passed. I made some changes, removed further empty props too,

Reply:
yeah I just meant the `transformerSource`. I think I may have had to highlight both because github was being weird.